packages/pangea-sdk/PangeaCyber.Net/PangeaCyber.Net.csproj

```diff
@@ -16,6 +16,7 @@
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
```

Comment:
What's this for?

Reply:
If you've tested the additions to work with .NET Framework v4.6.1, then we're good. Otherwise we may have to update our target frameworks.